ibm/mas_devops/roles/suite_switch_to_olm/README.md

```diff
@@ -0,0 +1,27 @@
+suite_switch_to_olm
+=============
+This role provides capabilities to switch a MAS instance from legacy deployment using mas-install.sh installer to OLM/Channel subscription install.
+
+Role Variables
+--------------
+- `mas_catalog_source` Defines the catalog to be used to install MAS channel subscription. Default to be used is `ibm-operator-catalog`.
```

Comment:
Can we convert these to the new (well not really new anymore) format for the the variable list

```
### var name
Description paragraph

- Required/Optional
- Environment Variable: `xxx`
- Default: `xxx`
```

Reply:
Fixed it